### DISCUSSION TITLE
Link to palettes

### DISCUSSION THREAD

**Discussion by heyakyra** [primer/prism#73] [Ideas]
https://github.com/adobe/leonardo supports this https://leonardocolor.io/theme.html

**Comment by lesliecdubs** [comment]
👋🏻 Hi @heyakyra, this is neat! Interested in opening a PR to Prism to add it?

**Reply by heyakyra** [reply]
If I can catch up on my backlog that would be great to work on, but no promises on whether I will be able to because it seems like a significant change (basically, each change to a palette should be reflected in the history/URL).

**Reply by lesliecdubs** [reply]
Understood. Feature requests for `primer/prism` are not currently on our roadmap so it's unlikely we'll work on this soon either, but we would accept a contribution if someone else wants to give it a go.
